tests/cephfs/cephfs_bugs/test_validate_mclientcaps_revoke_ack.py

```diff
@@ -0,0 +1,231 @@
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83584084 - Validate if no repeated mclientscaps(revoke) entries in MDS which
+                    was causing the Ceph Status to go Unhealthy
+
+    Args:
+        ceph_cluster (CephCluster): Ceph cluster object.
+        **kw: Additional keyword arguments.
+
+    Returns:
+        int: 0 if successful, 1 if failed.
+
+    Procedure:
+        1. Create 2 Subvolumes.
+        2. Write a large file on both the Subvolumes.
+        3. Validate the Ceph Status - should remain healthy.
+        4. Validate no mclientcaps(revoke) warnings seen in all active MDS Logs.
+    Clean-up:
+        1. Delete Subvolumes and Subvolumegroup
+    """
+    try:
+        tc = "CEPH-83584084"
+        log.info(f"Running cephfs {tc} test case")
+        fs_util_v1 = FsUtilsV1(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        mds_nodes = ceph_cluster.get_ceph_objects("mds")
+        log.info("checking Pre-requisites")
+        if len(clients) < 1:
+            log.info(
+                f"This test requires minimum 1 client nodes.This has only {len(clients)} clients"
+            )
+            return 1
+        fs_util_v1.prepare_clients(clients, build)
+        fs_util_v1.auth_list(clients)
+        default_fs = "cephfs"
+        subvolume_group_name = "subvol_group1"
+        subvolume_name = "subvol"
+
+        clients[0].exec_command(
+            sudo=True,
+            cmd="ceph config set mds log_to_file true",
+        )
+        out, rc = clients[0].exec_command(
+            sudo=True,
+            cmd="ceph config get mds log_to_file",
+        )
```

Comment:
This way of setting was not logging the things to log file on mds nodes Hemanth. 
Was this working now ?